paasta_tools/api/api_docs/swagger.json

```diff
@@ -652,6 +653,25 @@
           "format": "int32",
           "description": "The number of desired instances of the service"
         },
+        "autoscaling_info": {
```

Comment:
This is potentially where you could have shipped this incrementally, by adding one additional piece of data at a time (auto, status, mesos, ss)